Formula/ntopng.rb

```diff
@@ -1,9 +1,17 @@
 class Ntopng < Formula
   desc "Next generation version of the original ntop"
   homepage "http://www.ntop.org/products/ntop/"
-  url "https://downloads.sourceforge.net/project/ntop/ntopng/ntopng-2.2.tar.gz"
-  sha256 "4fccfc9e9f333addcd3c957b4520c471117bc2df5655d6eabf328c7385fb255e"
-  revision 1
+
+  stable do
+    url "https://github.com/ntop/ntopng/archive/2.4.tar.gz"
+    sha256 "86f8ed46983f46bcd931304d3d992fc1af572b11e461ab9fb4f0f472429bd5dd"
+
+    resource "nDPI" do
+      # tip of 1.8-stable branch; four commits beyond the 1.8 tag
+      url "https://github.com/ntop/nDPI.git",
+      :revision => "6fb81f146e2542cfbf7fab7d53678339c7747b35"
```

Comment:
So the short-term fix would be:

1. Make sure `gdm` no longer screws them up.

And the long-term fix could be:

1. Agree on the preferred style.
2. Re-implement `brew audit --strict` as an extension for RuboCop and enable it by default.
  1. Implement a cop for alignment of the `url` method arguments.
  2. Or just re-enable the existing default cop for method argument alignment.
3. Mass-change all formulae by using RuboCop's auto-fix feature.